c2corg_ui/static/js/editing/documentediting.js

```diff
@@ -242,6 +242,9 @@ app.DocumentEditingController.prototype.successRead = function(response) {
   if (this.modelName === 'image') {
     data['date_time'] = new Date(data['date_time']);
   }
+  if (this.modelName === 'xreport') {
+    data['date'] = new Date(data['date']);
+  }
```

Comment:
This should be moved to a ``filterData()`` method in ``app.XreportEditingController``. As in https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/editing/imageediting.js#L58-L69

This will also fix the conflict with this file.